File: RR_cpp/plot_mm1.py
from matplotlib import pyplot as plt
import pandas as pd
plt.subplot(1,2,1)
res=pd.read_csv("res_mm1.dat",sep="    ",header=None)
plt.plot(res[0],res[1])
plt.title("waitingTime")

plt.subplot(1,2,2)
res=pd.read_csv("len_mm1.dat",sep="    ",header=None)
plt.plot(res[0],res[1])
plt.title("queueLength")

plt.show()

File: RR_cpp/plot_mm1.m
clc;
close all;
m1=load("./res_output/res_mm1.dat");
subplot(1,2,1)
x1=m1(:,1);
y1=m1(:,2);
semilogy(x1,y1)
title('waitingTime')
subplot(1,2,2)
m2=load("./res_output/len_mm1.dat");
x2=m2(:,1);
y2=m2(:,2);
semilogy(x2,y2)
title('queueLength')